tutorials/notebooks/MCT_notebooks.md

```diff
@@ -1,30 +1,6 @@
-# Tutorials
-
-## Table of Contents
-- [Introduction](#introduction)
-- [Keras Tutorials](#keras-tutorials)
-  - [Post-Training Quantization](#keras-ptq)
-  - [Gradient-Based Post-Training Quantization](#keras-gptq)
-  - [Quantization-Aware Training](#keras-qat)
-  - [Structured Pruning](#keras-pruning)
-  - [Export Quantized Models](#keras-export)
-  - [Debug Tools](#keras-debug)
-- [Pytorch Tutorials](#pytorch-tutorials)
-  - [Quick-Start with Torchvision](#pytorch-quickstart-torchvision)
-  - [Post-Training Quantization](#pytorch-ptq)
-  - [Quantization-Aware Training](#pytorch-qat)
-  - [Structured Pruning](#pytorch-pruning)
-  - [Data Generation](#pytorch-data-generation)
-  - [Export Quantized Models](#pytorch-export)
-
-## Introduction
-Dive into the Model-Compression-Toolkit (MCT) with our collection of tutorials, covering a wide 
-range of compression techniques for Keras and Pytorch models. We provide
-both Python scripts and interactive Jupyter notebooks for an
-engaging and hands-on experience.
-
-
-## Keras Tutorials
+# MCT Features
+
```

Comment:
missing introduction on purpose? Is this my task on a separate PR? (because the rest of the page seems pretty much completed)